### DISCUSSION TITLE
Bumping open issues

### DISCUSSION THREAD

**Discussion by soerenwolfers** [duckdb/duckdb#12021] [General]
How do the devs feel about  "bumping" (via comments that just say "bump" or "+1") old open issues (especially those that haven't received any comments or labels from devs beyond "reproduced"  for more than a month)? I could see this sometimes being helpful in bringing attention to and separating old but relevant issues (that just happened to be overlooked at the time of creation) from other old issues that, e.g., are already solved but haven't been labeled so / were based on misunderstandings / were attempted but turned out too complex; but I at the same time I wouldn't want to appear impatient or artificially manipulate prioritization with meaningless comments

**Comment by Sin-Shadow-Fox** [comment]
I'd also like to know the answer to this. I know in most forums the rule of thumb is wait till your thread/discussion/issue is pushed off the first page but some collaborators on github seem to be a bit touchy.
